web/src/pages/indicators/inconsultation.page.tsx

```diff
@@ -1,9 +1,100 @@
 import React from "react";
 
-export default function InConsultationPage(): JSX.Element {
-	return (
-		<>
-			<p>Indicators in consultation</p>
-		</>
-	);
-}
+import { Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Document, SortOrder } from "@nice-digital/search-client";
+
+import { IndicatorListNav } from "@/components/ProductListNav/IndicatorListNav";
+import {
+	getProductListPage,
+	getGetServerSidePropsFunc,
+} from "@/components/ProductListPage/ProductListPage";
+import { ResponsiveDate } from "@/components/ResponsiveDate/ResponsiveDate";
+
+const defaultSortOrder = SortOrder.dateDescending,
+	dateFilterLabel = "Last updated date";
+
+const tableBodyRender = (documents: Document[]) => (
+	<>
+		<caption className="visually-hidden">
+			Published guidance, NICE advice and quality standards
+		</caption>
+		<thead>
+			<tr>
+				<th scope="col">Title</th>
+				<th scope="col">Reference number</th>
+				<th scope="col">Published</th>
```

Comment:
Wrong columns for consultations:
![image](https://user-images.githubusercontent.com/893676/201164204-a5131239-418a-4589-ad97-f26d75a8f1b8.png)